sklearn/model_selection/_search.py

```diff
@@ -314,20 +319,24 @@ def fit_grid_point(X, y, estimator, parameters, train, test, scorer,
     Returns
     -------
     score : float
-        Score of this parameter setting on given training / test split.
+         Score of this parameter setting on given training / test split.
 
     parameters : dict
         The parameters that have been evaluated.
 
     n_samples_test : int
         Number of test samples in this split.
     """
-    score, n_samples_test, _ = _fit_and_score(estimator, X, y, scorer, train,
-                                              test, verbose, parameters,
-                                              fit_params=fit_params,
-                                              return_n_test_samples=True,
-                                              error_score=error_score)
-    return score, parameters, n_samples_test
+    # NOTE we are not using the return value as the scorer by itself should be
+    # validated before. We use check_scoring only to reject multimetric scorer
```

Comment:
I'm not a I get this. You just mean that a scoring string is not supported?

fit_grid_point is a standalone public function which I consider obsolete and unmaintainable